coldcard/src/lib.rs

```diff
@@ -593,6 +630,18 @@ impl Coldcard {
         response.into_ascii().map(Some).map_err(Error::from)
     }
 
+    /// List miniscript descriptors registered on the device
+    pub fn miniscript_list(&mut self) -> Result<Vec<String>, Error> {
+        let resp = self.send(Request::MiniscriptList)?.into_ascii()?;
+        let resp = resp.replace("\"", "");
+        let end = resp.len() - 1;
+        let resp = resp[1..end].to_string();
+
+        let miniscripts: Vec<_> = resp.split(",").map(|s| s.trim().to_string()).collect();
```

Comment:
Is it possible to add a unit test that tests this parsing logic against an encoded response?